exercises/meetup/example.js

```diff
@@ -56,15 +53,17 @@ function _getDayIndex(day) {
     'saturday': 6
   };
 
-  day = day.toLowerCase();
+  var temp = day.toLowerCase();
```

Comment:
Hmm.  Can we find a more descriptive variable name than `temp` for this?  Otherwise, I'd rather see:

```javascript
return daysInd[day.toLowerCase()];
```